starter-code/javascript/main.js

```diff
@@ -1,60 +1,99 @@
 var chronometer = new Chronometer();
-var btnLeft     = document.getElementById('btnLeft');
-var btnRight    = document.getElementById('btnRight');
-var minDec      = document.getElementById('minDec');
-var minUni      = document.getElementById('minUni');
-var secDec      = document.getElementById('secDec');
-var secUni      = document.getElementById('secUni');
-var milDec      = document.getElementById('milDec');
-var milUni      = document.getElementById('milUni');
+var btnLeft = document.getElementById("btnLeft");
+var btnRight = document.getElementById("btnRight");
+var min = document.getElementById("min");
+var sec = document.getElementById("sec");
+var milDec = document.getElementById("milDec");
+var milUni = document.getElementById("milUni");
+var splits = document.getElementById("splits");
 
+//console.log(chronometer, btnLeft, btnRight, minDec, minUni, secDec, secUni, milDec, milUni)
 
-function printTime() {
 
-}
-
-function printMinutes() {
+const printTime = () => {
+    
+    const seconds = chronometer.twoDigitsNumber(chronometer.getSeconds())
+    printSeconds(seconds)
+    const minutes = chronometer.twoDigitsNumber(chronometer.getMinutes())
+    printMinutes(minutes)
 
 }
 
-function printSeconds() {
-
+function printMinutes(minutes) {
+    min.innerHTML = minutes;
 }
 
-function printMilliseconds() {
-
+function printSeconds(seconds) {
+    sec.innerHTML = seconds;
 }
 
-function printSplit() {
+function printMilliseconds() {}
 
+function printSplit() {
+    const seconds = chronometer.twoDigitsNumber(chronometer.getSeconds())
+    const minutes = chronometer.twoDigitsNumber(chronometer.getMinutes())
+    const li = document.createElement("li");
+    li.innerHTML = minutes + ":" + seconds;
+    splits.appendChild(li);
 }
 
 function clearSplits() {
-
+    splits.innerHTML = ""
 }
 
 function setStopBtn() {
-
+    chronometer.startClick();
+    btnLeft.innerHTML = "STOP";
+    btnLeft.className = "btn stop";
 }
 
 function setSplitBtn() {
-
+    btnRight.innerHTML = "SPLIT";
+    btnRight.className = "btn split";
 }
 
 function setStartBtn() {
-
+    chronometer.stopClick();
+    btnLeft.innerHTML = "START";  
+    btnLeft.className = "btn start";
 }
 
 function setResetBtn() {
-
+    btnRight.innerHTML = "RESET";
+    btnRight.className = "btn reset";
 }
 
+window.setInterval(printTime, 1000);
```

Comment:
You should make the interval a little faster than the chronometer otherwise the time is not updated according to the chronometers current time value, but a second after the startClick for example.

`window.setInterval(printTime, 100);`